Gemfile

```
@@ -21,6 +21,8 @@ gem "faker", "~> 2.14"
 
 gem "wicked_pdf", "~> 2.1"
 
+gem 'decidim-microsoft'
```

```suggestion
gem 'decidim-microsoft', "~ 1.0.0"
```

Why don't we specify the version number here?

---

ok, I will do it